lagrangebench/train/trainer.py

```diff
@@ -40,18 +52,17 @@ def _mse(
     particle_type: jnp.ndarray,
     target: jnp.ndarray,
     model_fn: Callable,
-    loss_weight: LossConfig,
+    loss_weight: Dict[str, float],
```

Comment:
Small thing but don't think this is the case. The `LossConfig` dataclass is passed

Reply:
Is this still relevant?